mobile/src/components/uam/groups-table/groups-table.tsx

```diff
@@ -44,7 +44,7 @@ const GroupsTable: FC = () => {
     <View style={styles.container}>
       <Text style={styles.title}>Groups</Text>
       <Table
-        columnWidthArr={[width * 0.15, width * 0.35, width * 0.2, width * 0.2]}
+        columnWidth={width * 0.4}
```

Comment:
Remove width, we just need to set fixed size that will looks good, like:
```suggestion
        columnWidthArr={[50, 250, 250, 150]}
```
Also do same for all the tables that we have, and remove useWindowDimensions hook from hooks/hooks